docs/DICTIONARIES.md

```diff
@@ -17,10 +17,12 @@ This project uses `dictionaries` to unify props between different components.
 
 ### Alignment
 
-| Dictionary | Values                    | Code name  |
-| ---------- | ------------------------- | ---------- |
-| AlignmentX | `left`, `center`, `right` | AlignmentX |
-| AlignmentY | `top`, `center`, `bottom` | AlignmentY |
+| Dictionary         | Values                               | Code name          |
+| ------------------ | ------------------------------------ | ------------------ |
+| AlignmentX         | `left`, `center`, `right`            | AlignmentX         |
+| AlignmentXExtended | `stretch`, `left`, `center`, `right` | AlignmentXExtended |
+| AlignmentY         | `top`, `center`, `bottom`            | AlignmentY         |
+| AlignmentYExtended | `stretch`, `top`, `center`, `bottom` | AlignmentYExtended |
```

Comment:
Nice! 👏🏻  I just wrote a Sass equivalent of this and it happens to look almost the same 🙂.

Just two things:

1. For the extended dictionaries, please use the same syntax as in the Size dictionary below, i.e. `AlignmentX + 'stretch'` (just with backticks).
2. The x-axis dictionary will be extended with `stretch` and `space-between`,
3. the y-axis dictionary will be extended with `stretch` and `baseline`.

Reply:
syntax updated